src/core/translation/locales/en/impact.json

```diff
@@ -0,0 +1,211 @@
+{
```

Comment:
please remove symbols and emojies and follow same rule everywhere.
still didn't get usage of `{}`.
please remove unnecessary/unsed things to have shorter json file (it is important to have small files)